commerce-subscription-web/src/main/java/com/liferay/commerce/subscription/web/internal/display/context/CommerceSubscriptionContentDisplayContext.java

```diff
@@ -179,7 +180,7 @@ public PortletURL getPortletURL() throws PortalException {
 	private final ConfigurationProvider _configurationProvider;
 	private final CPDefinitionHelper _cpDefinitionHelper;
 	private final CPInstanceHelper _cpInstanceHelper;
-	private final CPRequestHelper _cpRequestHelper;
+	private final CommerceSubscriptionRequestHelper _cpRequestHelper;
```

Comment:
`CommerceSubscriptionRequestHelper _cpRequestHelper` should be `CommerceSubscriptionRequestHelper _commerceSubscriptionRequestHelper`